src/code/db_camera.c

```diff
@@ -3,71 +3,75 @@
-char D_8012D070[] = "\x8CNﾌﾚ-ﾑ   \0\0\0\0";
-char D_8012D084[] = "\x8DZｶｲﾃﾝ   \0\0\0";
-char D_8012D098[] = "\x8Cﾓ-ﾄﾞ    \0\0\0\0";
-char D_8012D0AC[] = "  R\x8Dﾁｭｳｼﾝ  \0\0\0";
-
-char D_8012D0C0[] = "\x8DPｼﾞｶﾝ       \0\0";
-char D_8012D0D4[] = "\x8Dｷｮﾘ       \0";
-char D_8012D0E4[] = "\x8DXｶｲﾃﾝ       \0\0";
-char D_8012D0F8[] = "\x8DYｶｲﾃﾝ       \0\0";
+// TODO: cleanup these arrays and UB access
+char* D_8012CEE0[] = { GFXP_KATAKANA "ｷ-ﾌﾚ-ﾑ" GFXP_HIRAGANA "ｶﾞ" };
+char* D_8012CEE4 = GFXP_HIRAGANA "ﾀﾘﾏｾﾝ｡";
+char* D_8012CEE8 = GFXP_HIRAGANA "ｻｲｾｲﾃﾞｷﾏｾﾝ";
+char* D_8012CEEC = GFXP_HIRAGANA "ｻｲｾｲｼｭｳﾘｮｳ";
+char* D_8012CEF0 = GFXP_HIRAGANA "ｻｲｾｲﾁｭｳ!";
+
+char* D_8012CEF4 = "DEMO CAMERA TOOL";
+char* D_8012CEF8[] = { GFXP_HIRAGANA "ﾓｳﾊｲﾘﾏｾﾝ", GFXP_HIRAGANA "ﾄｳﾛｸ   ﾃﾝﾒ", GFXP_HIRAGANA "ﾍﾝｺｳ   /  " };
+char* D_8012CF04 = ">            >";
+char* D_8012CF08 = "<            <";
+char* D_8012CF0C = "<          >";
+char* D_8012CF10 = GFXP_KATAKANA "*ﾌﾟﾚｲﾔ-*";
+char* D_8012CF14 = GFXP_KATAKANA "Eﾓ-ﾄﾞ" GFXP_HIRAGANA " ｿｳﾀｲ";
+char* D_8012CF18[] = { GFXP_KATAKANA "Eﾓ-ﾄﾞ" GFXP_HIRAGANA "ｾﾞｯﾀｲ", GFXP_HIRAGANA "ｶﾞﾒﾝ" GFXP_KATAKANA "   ﾃﾞﾓ",
+                       GFXP_HIRAGANA "ｶﾞﾒﾝ   ﾌﾂｳ" };
+char* D_8012CF24[] = { GFXP_HIRAGANA "Pｼﾞｶﾝ  MAX", GFXP_KATAKANA "ﾘﾝｸ" GFXP_HIRAGANA "    ｷｵｸ",
+                       GFXP_KATAKANA "ﾘﾝｸ" GFXP_HIRAGANA "     ﾑｼ" };
+char* D_8012CF30 = GFXP_HIRAGANA "*ﾐﾃﾙｲﾁ*";
+char* D_8012CF34 = GFXP_KATAKANA "*ｶﾒﾗ" GFXP_HIRAGANA "ｲﾁ*";
+char* D_8012CF38 = "DEBUG CAMERA";
+char* D_8012CF3C = GFXP_KATAKANA "ｾﾝﾀ-/ﾛｯｸ";
+char* D_8012CF40 = GFXP_KATAKANA "ｾﾝﾀ-/ﾌﾘ-";
+
+char* D_8012CF44 = "DEMO CONTROL";
+char* D_8012CF48 = GFXP_KATAKANA "ﾒﾓﾘ" GFXP_HIRAGANA "ｶﾞﾀﾘﾏｾﾝ";
+char* D_8012CF4C = "p";
+char* D_8012CF50[] = { "e", "s", "l", "c" };
+
+char* D_8012CF60[] = { GFXP_KATAKANA "ﾒﾓﾘﾊﾟｯｸ" }; // "Mempak"
+char* D_8012CF64 = GFXP_KATAKANA "ｾｰﾌﾞ";          // "Save"
+char* D_8012CF68 = GFXP_KATAKANA "ﾛｰﾄﾞ";          // "Load"
+char* D_8012CF6C = GFXP_KATAKANA "ｸﾘｱ-";          // "Clear"
+char* D_8012CF70 = GFXP_HIRAGANA "ｦﾇｶﾅｲﾃﾞﾈ";
+
+char* D_8012CF74 = "FREE      BYTE";
+char* D_8012CF78 = "NEED      BYTE";
+char* D_8012CF7C = GFXP_KATAKANA "*ﾒﾓﾘ-ﾊﾟｯｸ*";
+char* D_8012CF80 = GFXP_HIRAGANA "ｦﾐﾂｹﾗﾚﾏｾﾝ";
+char* D_8012CF84 = GFXP_KATAKANA "ﾌｧｲﾙ " GFXP_HIRAGANA "ｦ";
+char* D_8012CF88[] = { GFXP_HIRAGANA "ｼﾃﾓｲｲﾃﾞｽｶ?", GFXP_HIRAGANA "ｹﾞﾝｻﾞｲﾍﾝｼｭｳﾁｭｳﾉ",
+                       GFXP_KATAKANA "ﾌｧｲﾙ" GFXP_HIRAGANA "ﾊﾊｷｻﾚﾏｽ" };
+char* D_8012CF94 = GFXP_HIRAGANA "ﾊｲ";
+char* D_8012CF98 = GFXP_HIRAGANA "ｲｲｴ";
+char* D_8012CF9C[] = { GFXP_HIRAGANA "ｼﾃｲﾏｽ", GFXP_HIRAGANA "ｳﾜｶﾞｷ" };
+char* D_8012CFA4 = GFXP_HIRAGANA "ｼﾏｼﾀ";
+char* D_8012CFA8 = "USE       BYTE";
+char* D_8012CFAC = GFXP_HIRAGANA "ﾆｼｯﾊﾟｲ";
+
+char* D_8012CFB0 = GFXP_KATAKANA "Eﾓ-ﾄﾞ" GFXP_HIRAGANA "  ｺﾃｲ";
+char D_8012CFB4[] = GFXP_KATAKANA "ﾌﾚ-ﾑ       ";
+char D_8012CFC4[] = GFXP_KATAKANA "ｷ-   /   ";
+char D_8012CFD0[] = GFXP_HIRAGANA "(ｾﾝﾀ-ﾃﾝ)\0\0\0\0\0";
```

Comment:
Apparently they are there on purpose and get replaced at runtime according to random in a previous discord discussion (https://discord.com/channels/688807550715560050/688851317593997489/860439053576830976)

But overall I'm not quite sure what's going on with all these strings tbh, which is part of why I left a TODO there. It will need its own dedicated analysis to figure out how they should really be laid out.